src/tools/generate/generate_chapters.js

```diff
@@ -66,20 +77,24 @@ const generate_chapters = async (chapter_match) => {
 
       const markdown = await fs.readFile(file, 'utf-8');
       const { metadata, body, toc } = await parse_file(markdown,chapter);
-      const chapter_featured_quote = generate_chapter_featured_quote(metadata);
-      if (Object.keys(chapter_featured_quote).length > 0) {
-        if (!(language in featured_quotes)) {
-          featured_quotes[language] = {};
+
+      // Only included "done" chapters in featured quotes, sitemaps...etc.
+      if (!chapter_config[year][chapter].todo) {
+        const chapter_featured_quote = generate_chapter_featured_quote(metadata);
+        if (Object.keys(chapter_featured_quote).length > 0) {
+          if (!(language in featured_quotes)) {
+            featured_quotes[language] = {};
+          }
+          if (!(year in featured_quotes[language])) {
+            featured_quotes[language][year] = {};
+          }
+          featured_quotes[language][year][chapter] = chapter_featured_quote;
         }
-        if (!(year in featured_quotes[language])) {
-          featured_quotes[language][year] = {};
+        if ( sitemap_languages[year].includes(language) ) {
+          sitemap.push({ language, year, chapter, metadata });
         }
-        featured_quotes[language][year][chapter] = chapter_featured_quote;
-      }
-      if ( sitemap_languages[year].includes(language) ) {
-        sitemap.push({ language, year, chapter, metadata });
+        ebook_chapters.push({ language, year, chapter, metadata, body, toc });
       }
```

Comment:
This is all just in indentation change - apart from the first `if` statement. So looks a lot more than it is.